code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -196,6 +196,15 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/weapon/wrench(src)
 	src.modules += new /obj/item/weapon/crowbar(src)
 	src.modules += new /obj/item/device/healthanalyzer(src)
+	src.modules += new /obj/item/weapon/reagent_containers/glass/bucket(src) // a hydroponist's bucket 
+	src.modules += new /obj/item/weapon/material/minihoe(src)  // a hydroponist's hoe 
+	src.modules += new /obj/item/weapon/material/kitchen/utensil/knife/plastic(src) // a plastic kitchen knife 

```

Comment:
That better fits into the service module (if it doesn't have something similar already)